app/pipeline/chat_gpt_wrapper_pipeline.py

```
@@ -0,0 +1,96 @@
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)